src/Dev/State/LoggerTestState.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SilverStripe\MFA\Dev\State;
+
+use Monolog\Handler\NullHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Dev\State\TestState;
+
+/**
+ * Clears any user defined loggers during unit test execution
+ */
+class LoggerTestState implements TestState
+{
+    public function setUp(SapphireTest $test)
+    {
+        /** @var Logger $logger */
+        $logger = Injector::inst()->get(LoggerInterface::class . '.mfa');
+        $logger->setHandlers([new NullHandler()]);
+    }
+
+    public function tearDown(SapphireTest $test)
+    {
+        // noop
+    }
+
+    public function setUpOnce($class)
```

Comment:
What is the point of this method (and `tearDownOnce`)?
As far as I know these interfaces are defined neither in PHPUnit nor SapphireTest

Reply:
They're methods in `TestState`

Reply:
Ohh, I've completely misread the purpose of this class 😂